Extension/src/LanguageServer/client.ts

```diff
@@ -2724,15 +2728,22 @@ export class DefaultClient implements Client {
 
         showConfigStatus = showConfigStatus || (configurationNotSet &&
             !!compilerDefaults && !compilerDefaults.trustedCompilerFound && trustedCompilerPaths && (trustedCompilerPaths.length !== 1 || trustedCompilerPaths[0] !== ""));
+        if (showConfigStatus) {
+            this.showConfigureIntelliSenseButton = true;
+        } else {
+            this.showConfigureIntelliSenseButton = false;
+        }
+        if (!configProviderNotSet) {
+            ui.ShowConfigureIntelliSenseButton(this.showConfigureIntelliSenseButton, this, ConfigurationType.ConfigProvider, "handleConfig");
+        } else if (!compileCommandsNotSet) {
+            ui.ShowConfigureIntelliSenseButton(this.showConfigureIntelliSenseButton, this, ConfigurationType.CompileCommands, "handleConfig");
+        } else if (!compilerPathNotSet) {
+            ui.ShowConfigureIntelliSenseButton(this.showConfigureIntelliSenseButton, this, ConfigurationType.CompilerPath, "handleConfig");
```

Comment:
This ConfigurationType.CompilerPath also occurring when compilerPath is "" instead of NotConfigured.

Reply:
...or is that intentional? I was seeing the opposite in the configurationChange handler, so it was sending inconsistent results for the same config.

Reply:
`compilerPathNotSet` is only testing for `undefined`.  If it's `""`, that should count as configured.